client/components/Byline/Byline.tsx

```diff
@@ -50,6 +58,45 @@ const Byline = (props: PrivateBylineProps) => {
 		truncateAt = null,
 	} = props;
 
+	let bylinePrefixObject;
+
+	switch (bylinePrefix) {
+		case BylineOption.By:
+			bylinePrefixObject = BylineOption.By;
+			break;
+		case BylineOption.Contributors:
+			bylinePrefixObject = (
+				<Tooltip
+					content={BylineOption.Contributors}
+					children={<Icon iconSize={14} icon="people" />}
+					className="byline-icon"
+				/>
+			);
+			break;
+		case BylineOption.Az:
+			bylinePrefixObject = (
+				<Tooltip
+					content={BylineOption.Az}
+					children={
+						<Icon iconSize={14} icon="sort-alphabetical" className="byline-icon" />
+					}
+				/>
+			);
+			break;
+		case BylineOption.Za:
+			bylinePrefixObject = (
+				<Tooltip
+					content={BylineOption.Za}
+					children={
+						<Icon iconSize={14} icon="sort-alphabetical-desc" className="byline-icon" />
+					}
+				/>
+			);
+			break;
+		default:
+			bylinePrefixObject = 'by';
+			break;
+	}
```

Comment:
This might break any existing `bylinePrefix`es used today that aren't "by", since this overwrites the `string` variant of the `bylinePrefix` prop with "by" if it doesn't match one of the `BylineOption` values.